exporter/elasticsearchexporter/utils_test.go

```diff
@@ -26,6 +28,29 @@ type itemRequest struct {
 	Document json.RawMessage
 }
 
+func itemRequestsSortFunc(a, b itemRequest) int {
+	comp := bytes.Compare(a.Action, b.Action)
+	if comp == 0 {
+		return bytes.Compare(a.Document, b.Document)
+	}
+	return comp
+}
+
+func assertItemsEqual(t *testing.T, expected, actual []itemRequest, assertOrder bool) {
```

Comment:
Probably a followup, but seems like we could get by with `assert.ElementsMatch` if we were to convert the raw JSON to strings, and compare those. I expect the error messages would be more useful then too.